lib/resources/dimensions.dart

```diff
@@ -25,4 +25,6 @@ class Dimensions {
   static const double homeSkeletonLoadingTextBorderRadius = 10.0;
 
   static const double answerEmojiSize = 34.0;
+
+  static const double answerNpsDividerWidth = 1.0;
```

Comment:
![Screenshot 2566-07-10 at 10 37 18 AM](https://github.com/nimblehq/flutter-ic-lydia-ryan/assets/12026942/81db01db-aa12-421b-aee1-80276ee3f050)

@ryan-conway We use `0.5` in the design. And this is the same value for another divider too. Should we apply `1` dp to all dividers? Or should we update this value to be `0.5` and name it general like `dividerWidth` as you did before?

Reply:
I like the general idea, updated in [ef138a6](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/93/commits/ef138a62513adb3765f5d8f316fcc3d98e6ff196)